docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -107,6 +107,41 @@ service:
       exporters: [..., prometheusremotewrite]
 ```
 
+## Work with default OpenTelemetry labels
+
+OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
+
+To optimize the storage of and ability to query this data, Mimir automatically assigns the following OTel resource attributes as labels at the time of ingestion, with periods (`.`) replaced by underscores (`_`):
+
+- `service.instance.id`
+- `service.name`
+- `service.namespace`
+- `service.version`
+- `cloud.availability_zone`
+- `cloud region`
+- `container.name`
+- `deployment.environment.name`
+- `k8s.cluster.name`
+- `k8s.container.name`
+- `k8s.cronjob.name`
+- `k8s.daemonset.name`
+- `k8s.deployment.name`
+- `k8s.job.name`
+- `k8s.namespace.name`
+- `k8s.pod.name`
+- `k8s.replicaset.name`
+- `k8s.statefulset.name`
+
+If you are using Grafana Cloud, contact support to disable this setting or to update this list.
+
+{{< admonition type="note" >}}
+Some of these labels are mutually exclusive. If you don't need a particular label, you can aggregate it using Adaptive Metrics. For more information about Adaptive Metrics, refer to [Reduce metrics costs via Adaptive Metrics](https://grafana.com/docs/grafana-cloud/cost-management-and-billing/adaptive-telemetry/adaptive-metrics/).
```

Comment:
Does the note refer to the list of _resource attributes_ above (as opposed to labels)? Do you have examples of mutually exclusive ones? Not sure how to interpret the note.

If you are referring to resource attributes, why not avoid promoting them instead of aggregating the resulting labels away?

Reply:
Good catch, they are resource attributes. I actually think this note might add more confusion than clarity, and I'm going to remove it.